.github/workflows/build_book.yaml

```diff
@@ -1,57 +1,60 @@
-# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
-# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
   push:
-    branches: [main, master]
+    branches: main
   pull_request:
-    branches: [main, master]
+    branches: main
+  # to be able to trigger a manual build
+  workflow_dispatch:
   schedule:
     # run every day at 11 PM
     - cron: '0 23 * * *'
 
-name: bookdown
+name: Render and deploy Book to Netlify
+
+env:
+  isExtPR: ${{ github.event.pull_request.head.repo.fork == true }}
+  RUST_BACKTRACE: 1
 
 jobs:
-  bookdown:
+  build-deploy:
     runs-on: ubuntu-latest
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
-      isExtPR: ${{ github.event.pull_request.head.repo.fork == true }}
-
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v2
 
-      - uses: r-lib/actions/setup-pandoc@v2
+      - name: Install Quarto
+        uses: quarto-dev/quarto-actions/install-quarto@v1
+        with:
+          # To install LaTeX to build PDF book
+          tinytex: true
+          # uncomment below and fill to pin a version
+          # version: 0.9.105
 
       - uses: r-lib/actions/setup-r@v2
         with:
           use-public-rspm: true
 
       - uses: r-lib/actions/setup-r-dependencies@v2
 
-      - name: Cache bookdown results
```

Comment:
Did you deliberately drop the caching? I have no idea if it's important or not.

Reply:
Semi-deliberate. I'm not entirely sure what (if any) files need to be cached in the quarto book, and since there's no caching used in the r4ds book workflow (which I was using as a guide) I didn't try to add anything. Maybe we leave it as is and try restoring the cache later if we have build issues?

Reply:
Having now merged I can see why we had a cache. The actions workflow takes about 20 mins to finish and I had timeout issues the first time. I'll open an issue to restore the cache and see what needs to happen